website/docs/i18n/i18n-tutorial.md

```diff
@@ -258,6 +258,52 @@ You can see the calls to the translation APIs as purely _markers_ that tell Docu
 
 :::
 
+#### Pluralization {#pluralization}
+
+When you run `write-translations`, you will notice that some labels are pluralized:
+
+```json title="i18n/en/code.json"
+{
+  // ...
+  "theme.blog.post.plurals": "One post|{count} posts"
+  // ...
+}
+```
+
+Every language will have a list of [possible plural categories](https://unicode-org.github.io/cldr-staging/charts/37/supplemental/language_plural_rules.html). Docusaurus will arrange them in the order of `["zero", "one", "two", "few", "many", "other"]`. For example, because English (`en`) has two plural forms ("one" and "other"), the translation message has two labels separated by a pipe (`|`). For Polish (`pl`) which has three plural forms ("one", "few", and "many"), you would provide three labels in that order, joined by pipes.
+
+You can pluralize your own code's messages as well:
+
+```jsx
+import {translate} from '@docusaurus/Translate';
+import {usePluralForm} from '@docusaurus/theme-common';
+
+function ItemsList({items}) {
+  // `usePluralForm` will provide the plural selector for the current locale
+  const {selectMessage} = usePluralForm();
+  // Select the appropriate pluralized label based on `items.length`
+  const message = selectMessage(
+    items.length,
+    translate(
+      {message: 'One item|{count} items'},
+      {count: items.length},
+    ),
+  );
```

Comment:
hmmm, this is what we have in our code but that looks a bit weird to me, as we translate everything and only then select

At the same time we have to pass a static message config to `translate()` otherwise it wouldn't be extracted

Maybe we'd need like react-intl a new method like `defineMessage()` that permits static extraction but without directly translating 🤷‍♂️ 

Not a big deal, we can keep it this way for now